app/Clock/ClockServiceProvider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Clock;
+
+use CultuurNet\UDB3\Container\AbstractServiceProvider;
+use DateTimeZone;
+
+final class ClockServiceProvider extends AbstractServiceProvider
+{
+    protected function getProvidedServiceNames(): array
+    {
+        return [
+            'timezone',
+            'clock',
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $container->addShared(
+            'timezone',
+            fn () => new DateTimeZone(
+                empty($container->get('config')['timezone']) ? 'Europe/Brussels' : $container->get('config')['timezone']
```

Comment:
At the top of `bootstrap.php` we also set the default timezone to a hardcoded value:
https://github.com/cultuurnet/udb3-silex/blob/621378c54ba6af1aed2a9593ef747c10e4ced630/bootstrap.php#L75

Maybe we should also use this `timezone` container value for that? (We'd need to move the `date_default_timezone_set` down a little bit but that should be okay)

Of course not very urgent/important because no one else is running this app in another timezone anyway at the moment.